test/test_dht.cpp

```diff
@@ -2692,6 +2704,39 @@ TORRENT_TEST(node_id_bucket_distribution)
 	}
 }
 
+TORRENT_TEST(node_id_min_distance_exp)
+{
+	node_id n1 = to_hash("0000000000000000000000000000000000000002");
+	node_id n2 = to_hash("0000000000000000000000000000000000000004");
+	node_id n3 = to_hash("0000000000000000000000000000000000000008");
+
+	std::vector<node_id> ids;
+
+	TEST_EQUAL(min_distance_exp(sha1_hash::min(), ids), 0);
```

Comment:
as I pointed out, this seems like a questionable use case. does it happen in real-life?